Makefile

```diff
@@ -23,7 +23,7 @@ test-unit:
 .PHONY: test-acceptance
 test-acceptance:
 	docker-compose -f compose/base.yml pull && docker-compose \
-	-p terraform-provider-mezmo-$(shell echo $$BUILD_TAG | tr '[:upper:]' '[:lower:]') \
+	-p terraform-provider-mezmo-$(shell echo $$BUILD_TAG | tr '[:upper:]' '[:lower:]' | sed s/%2f/_/g) \
```

Comment:
could also be

https://github.com/answerbook/logdna-web/commit/57d42ccffc838236e3c6f73cd03604ed99b46d46

Reply:
is there a preferable way? (real question)

Reply:
It will handle other "invalid" characters in the compose cluster name, but we don't usually use any other ones but `/` in branch names, so  ¯\\\_(ツ)_/¯ dealer's choice?

Reply:
ah that's a good point, let me go with the safer way